### ISSUE TITLE
NullReferenceException

### ISSUE THREAD

**Issue by ElweThor** [blaxxun-boop/Backpacks#2]
I may be wrong but, after the last ExtendedItemDataFramework update, every time I try to open Backpacks I receive this message in the log:
**NullReferenceException
  at (wrapper managed-to-native) UnityEngine.Component.get_transform(UnityEngine.Component)
  at Backpacks.CustomContainer+OpenFakeItemsContainer.Open (InventoryGui invGui, ItemDrop+ItemData item) [0x00079] in <f649525c84d44d5a8832b40d6118edee>:0 
  at Backpacks.CustomContainer+OpenFakeItemsContainer.Prefix (InventoryGui __instance) [0x00071] in <f649525c84d44d5a8832b40d6118edee>:0 
  at (wrapper dynamic-method) InventoryGui.DMD<InventoryGui::UpdateContainer>(InventoryGui,Player)
  at (wrapper dynamic-method) InventoryGui.DMD<InventoryGui::Update>(InventoryGui)**

And, when I try to close it, I can't, and receive this other message:
**NullReferenceException
  at (wrapper managed-to-native) UnityEngine.Component.get_transform(UnityEngine.Component)
  at Backpacks.CustomContainer+CloseFakeItemContainer.Prefix (InventoryGui __instance) [0x00009] in <f649525c84d44d5a8832b40d6118edee>:0 
  at (wrapper dynamic-method) InventoryGui.DMD<InventoryGui::Hide>(InventoryGui)
  at (wrapper dynamic-method) InventoryGui.DMD<InventoryGui::Update>(InventoryGui)**

At that point I had **no way to close** Backpacks and to play the game (maybe there is some trick to close it but didn't find it at now), as ESC don't even work, same as pressing E or TAB, so I've to terminate Valheim's task to brutally exit the game (not the very best, for opened files).

I repeat, it seems to me such behavior started happening after the abovementioned update, but that's my feeling only: I saw such behavior starting, at some point, and the only difference was that update. Before than that, Backpacks was working very well.

_As a side note, for the same reason I had to disable Azumatt's AAA_Crafting, which was creating problems towards AUGA but only after the same update I told._

I know Backpacks is in its early development stage, this bugrep is just for you to know.

**Comment by ElweThor** [comment]
I've seen RandyKnapp updated ExtendedItemDataFramework, so I re-tested Backpacks, but no way still: same NRE when trying to  close it.

**Comment by blaxxun-boop** [comment]
Backpacks do not need ExtendedItemDataFramework. It's an obsolete mod anyway and no mod should use it, since you can use IronGates native system for this now.
Not sure if Backpacks is compatible with ExtendedItemDataFramework. Might also be caused by a UI mod like AUGA.

**Comment by kil0byt3** [comment]
EpicLoot seems to still require EIDF, which is likely why it is installed for the above users

I can reproduce this issue as stated

**Comment by ElweThor** [comment]
@blaxxun-boop  I can confirm your statements:
1.  with a further check I understood EIDF wasn't used by any of the mods I'm running (probably a leftover from some mod I used in the past), and have no role into NRE.
2. following your words I disabled AUGA (which is a nice to have, but far less needed than Backpacks) and your Mod is giving no errors more: it seems it's definitely an AUGA problem.
Thanks for the info (and for the patience)
